src/components/ui_kit/photo_picker/mod.rs

```diff
@@ -49,8 +49,14 @@ pub fn PhotoPicker(cx: Scope<Props>) -> Element {
                     let path_image = FileDialog::new().add_filter("image", &["jpg", "png", "jpeg"]).set_directory(".").pick_file();
                     match path_image {
                         Some(path) => {
-                            let base64_image = to_base64(path.to_str().unwrap());
-                            match account.write().update_identity(IdentityUpdate::set_graphics_picture(base64_image)) {
+                            
+                            let file = std::fs::read(&path).unwrap();
```

Comment:
I wouldnt recommend unwrapping. Could use a match condition to be on the safe side.